03 Design/OCs/OCGetDoors.md

```diff
@@ -0,0 +1,19 @@
+#### Operation: 
+
+getDoors() 
+
+#### Cross reference: 
+
+Use case 1: Send Request 
+
+#### Preconditions: 
+
+Wall wall exists.
+
```

Comment:
Opdel List wall.listOfdoors of list  
List listOfDoors exists.
wall.listOfDoors exists.